src/UI/Factory.php

```diff
@@ -336,4 +337,27 @@ public function panel();
 	 **/
 	public function modal();
 
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *     Popovers are used to display secondary information to a component.
+	 *   composition: >
+	 *     Popovers consist of a short title and text.
```

Comment:
This is not sufficient. What about: "Popovers consist of a layer displayed above all other content. The exact layout of the popover heavily depends on the functionality it performs. However all Popovers contain a pointer pointing from the Popover body to the Trigger of the Popover."